content/en/docs/concepts/data-collection.md

```diff
@@ -10,17 +10,17 @@ removes the need to run, operate, and maintain multiple agents/collectors in
 order to support open-source observability data formats (e.g. Jaeger,
 Prometheus, etc.) sending to one or more open-source or commercial back-ends.
 In addition, the Collector gives end-users control of their data. The Collector
-is the default location for instrumentation libraries to send their telemetry
+is the default location instrumentation libraries export their telemetry
 data.
 
 ## Deployment
 
 The OpenTelemetry Collector provides a single binary and two deployment methods:
```

Comment:
worth mentioning what the "distro" means and what packages can be deployed

Reply:
I plan to address this one separately -- opened #381